examples/04_fuzzy_joining.py

```diff
@@ -416,9 +399,12 @@
 from sklearn.model_selection import GridSearchCV
 
 # We will test four possible values of match_score:
-params = {"joiner__match_score": [0.2, 0.3, 0.4, 0.5]}
+params = {
+    f"{joiner}__match_score": [0.2, 0.9]
+    for joiner in ["gdppc", "life_exp", "legal_rights"]
```

Comment:
Can we have a simpler (more tedious) syntax here, expanding the dict comprehension manually. It will be more readable for people who are not well versed in Python